### ISSUE TITLE
Paper.js causes exception of dispatchevent for any version higher than 0.9.8

### ISSUE THREAD

**Issue by feature-engineer** [paperjs/paper.js#1832]
# Description/Steps to reproduce

When trying to initiate an even dispatch, an exception is thrown instead of the event getting dispatched.

Run the following code:

```javascript
paper.install(window);
window.dispatchEvent(new Event('resize'));
```

# Link to reproduction test-case

Here's a minimal working example:
version 0.9.8 (works):
http://jsfiddle.net/6bcgpvto/31/

version 0.9.10 (all the higher versions up to latest also do the same - throw exception):
http://jsfiddle.net/6bcgpvto/32/

# Expected result

Not to raise any exception and handle the resizes as if they were user initiated.. 

Got the following exception:

> Failed to execute 'dispatchEvent' on 'EventTarget': parameter 1 is not of type 'Event'.

# Additional information
Pure JS, no node, reproducible on firefox and chrome.

**Comment by lehni** [comment]
Yes, this is because `paper.install()` overrides the native `Event` constructor with its own.

The change happened in 2013:

https://github.com/paperjs/paper.js/commit/9c98ef831638e8ea04681348450d382c41f53fe9#diff-639b76984f4eca3ca22752eb52883060

It's better to not relay on `paper.install()` and import the constructors that you want to use directly from `paper`.

**Comment by andyg2** [comment]
> Yes, this is because `paper.install()` overrides the native `Event` constructor with its own.
> 
> The change happened in 2013:
> 
> [9c98ef8#diff-639b76984f4eca3ca22752eb52883060](https://github.com/paperjs/paper.js/commit/9c98ef831638e8ea04681348450d382c41f53fe9#diff-639b76984f4eca3ca22752eb52883060)
> 
> It's better to not rely on `paper.install()` and import the constructors that you want to use directly from `paper`.

It seems this can be avoided by installing paper to an alternative variable e.g. a global.

```js
var myglobal; // global scope

// anywhere
paper.install(myglobal);
```

But you need to prefix every reference to paper with that 
```js
let polyPath = new myglobal.paper.Path();
```

There may well be a better way to do this but after a few days pulling out hair I would have accepted anything.
